History_CLI.txt

```diff
@@ -4,26 +4,32 @@ Notes:
 - Deleted
 x Correction
 
-Version 1.3.8, 2019-04-26
--------------
-
-Version 1.3.7, 2019-04-19
--------------
-+ Fix crash with empty fields in XML input
-+ Link between ISRC in aXML and ISRC in INFO (EBU recommendation)
-
-Version 1.3.6, 2018-09-10
--------------
-+ Loudness CLI parameters
-
-Version 1.3.5, 2018-08-27
--------------
-x Fix missing Null Character on LIST-INFO chunks during file writing
-
-Version 1.3.3, 2017-11-21
--------------
-x Fix regression preventing to store bext chunck version 0 or 1
-
+Version 1.4.0, 2020-05-20

```

Comment:
You forgot to update this line ;-).
(version + date + line about the version numbering)